website/docs/commands/repl.md

```diff
@@ -147,20 +174,54 @@ scala-cli repl --toolkit default
 ```
 
 ```text
-Welcome to Scala 3.3.1 (17, Java OpenJDK 64-Bit Server VM).
+Welcome to Scala 3.7.4 (17, Java OpenJDK 64-Bit Server VM).
 Type in expressions for evaluation. Or try :help.
                                                                                 
-scala> import os._
+scala> import os.*
                                                                                 
 scala> os.pwd
-val res0: os.Path = /Users/yadukrishnan/test
+val res0: os.Path = /current/directory/path
 scala> :quit
 
 ```
 </ChainedSnippets>
 
 Since we started the repl with toolkit enabled, we can use the libraries included in the toolkit directly. In the above example, the `os-lib` library from the toolkit is used to print the current path. 
 
+### Running snippets directly via the REPL
+It is possible to run code snippets via the REPL (with all the internal quirks of the REPL), rather than using standard runner execution.
```

Comment:
```suggestion
It is possible to run code snippets via the REPL, with all the internal quirks of the REPL, rather than using standard runner execution.
```
Do we need parenthesis?